examples/flava/train.py

```diff
@@ -41,21 +41,29 @@ def main():
     datamodule = MultiDataModule(datamodules)
 
     datamodule.setup("fit")
-    model = FLAVAPreTrainingLightningModule(
-        learning_rate=config.training.learning_rate,
-        adam_eps=config.training.adam_eps,
-        adam_weight_decay=config.training.adam_weight_decay,
-        adam_betas=config.training.adam_betas,
-        warmup_steps=config.training.warmup_steps,
-        max_steps=config.training.lightning.max_steps,
-        **config.model,
-    )
+    if config.training.lightning_load_from_checkpoint is None:
+        model = FLAVAPreTrainingLightningModule(
+            learning_rate=config.training.learning_rate,
+            adam_eps=config.training.adam_eps,
+            adam_weight_decay=config.training.adam_weight_decay,
+            adam_betas=config.training.adam_betas,
+            warmup_steps=config.training.warmup_steps,
+            max_steps=config.training.lightning_trainer.max_steps,
+            **config.model,
+        )
+    else:
+        model = FLAVAPreTrainingLightningModule.load_from_checkpoint(
+            config.training.lightning_load_from_checkpoint
+        )
 
     trainer = Trainer(
-        **OmegaConf.to_container(config.training.lightning),
+        **OmegaConf.to_container(config.training.lightning_trainer),
         callbacks=[
             LearningRateMonitor(logging_interval="step"),
             MultimodalEvalCallback(imagenet_datamodule=imagenet_datamodule),
+            ModelCheckpoint(
```

Comment:
how does this behave when checkpoint conf is empty dict? we dont want to start checkpointing on every iteration. can u double check

Reply:
I have made it completely no-op for the case when `config.training.lightning_checkpoint:` is missing.
It will be useful to have model checkpointing functionality fully unplugged by config change if some error will be inside that logic on pytorch lightning / pytorch side.